design/cryptography.md

```diff
@@ -120,8 +134,8 @@ An example would be:
 - Validators do not release keys, only the centralized sequencer releases them
 
 4. In the event of a db failure how does the central sequencer know which keys to recalculate
```

Comment:
I don't quite follow this

Reply:
Question on what happens if the stored revelation keys are lost. Reworded it to be clearer